### PR TITLE
Added + extension for AttributedStrings

### PR DESCRIPTION
**Pull request by lfarah**
## Checklist

- [x] New Extension
- [ ] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

File: Sources/NSAttributedStringExtensions.swift [hunk 1]
```diff
@@ -73,5 +73,12 @@ public func += (left: inout NSAttributedString, right: NSAttributedString) {
     ns.append(right)
     left = ns
 }
+    
+/// EZSE: Sum of one NSAttributedString with another NSAttributedString
+public func + (left: NSAttributedString, right: NSAttributedString) -> NSAttributedString {
```
Comment (Khalian) [inline Sources/NSAttributedStringExtensions.swift]: Does NSAttributedString support any other operators by default. If yes, then what is the operator precedence of those operators wrt to this +?

Comment (lfarah) [inline Sources/NSAttributedStringExtensions.swift]: Yes. It [does currently support **+=**](https://github.com/goktugyil/EZSwiftExtensions/blob/adding_attributed_string_extension/Sources/NSAttributedStringExtensions.swift#L71)

Comment (Khalian) [inline Sources/NSAttributedStringExtensions.swift]: Its not really about adding something. I like the extension, I am just confirmed about operator precedence. 

Ideally + should be higher precedence than +=.

For instance str1 += str2 + str3 should evaluate to str1 = (str1 + (str2 + str3))). I do not want the assignment to be done prior to the +. 

I wanted to verify that this behavior is consistent for this.

Comment (lfarah) [reply Sources/NSAttributedStringExtensions.swift]: I added one more test. What do you think?

Comment (Khalian) [reply Sources/NSAttributedStringExtensions.swift]: yea, this verifies my assumption. I will approve as soon as it passes the build.

**Comment by lfarah** [comment]
@Khalian please merge this and I'll fix the Swiftlint error in another PR

**Pull request merged**